File: Szulo.java
class Parent {

	void parent() {
	System.out.println("Ez a szülő osztály");
	}
}

class Gyerek extends Parent {
	void gyerek() {
	System.out.println("Ez a gyerek osztály");
	}
}

class Szulo {
	public static void main(String[] args){
	Parent something = new Gyerek();

	something.parent();
	something.gyerek();
	}
	
}
	
	

File: Szulo.cpp
#include <iostream>

using namespace std;

class Parent {
public : 
	void parent() {
	std::cout << " Ez a szülő osztály " << std::endl;
	}
};

class Gyerek : public Parent {
public :
	void gyerek() {
	std::cout << " Ez a gyerek osztály " << std::endl;
	}
};

int main(){
	Parent* something = new Gyerek();
	something->parent();
	something->gyerek();
	}
